tests/impact/client/fixtures.py

```diff
@@ -1004,10 +1004,7 @@ def workspace():
     exp_service.experiment_execute.return_value = "pid_2009"

```

Comment:
This file need the import:
`import modelon.impact.client.options`
to fix some lint errors